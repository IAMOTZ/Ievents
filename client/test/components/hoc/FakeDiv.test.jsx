import React from 'react';
import FakeDiv from '../../../components/hoc/FakeDiv';

describe('<FakeDiv />', () => {
  describe('Rendering:', () => {
    it('should render correctly', () => {
      const childComponent = <p>I am wrapped by a fake div</p>;
      const wrapper = shallow(<FakeDiv>{childComponent}</FakeDiv>);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
