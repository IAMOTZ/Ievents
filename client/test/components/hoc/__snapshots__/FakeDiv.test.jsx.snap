// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FakeDiv /> Rendering: should render correctly 1`] = `
<p>
  I am wrapped by a fake div
</p>
`;
